lessons/pydata/pandas_core/index.ipynb

```diff
@@ -0,0 +1,3402 @@
+{
```

Comment:
<p>Obecně bych dával komentáře až na výjimky do md buněk, ne do <code># komentářů</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyDataCZ/naucse.python.cz/pull/18/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
Mě ty komentáře takhle vyhovují, jak jsou vždy ke konkrétnímu řádku.

Reply:
Tak to nechme `# takhle :)`